### PR TITLE
add configmap support

### PR DESCRIPTION
**Pull request by missedone**
this PR add support for watching on configmap events

### CODE REVIEW

File: pkg/controller/controller.go [hunk 1]
```diff
@@ -293,6 +293,28 @@ func Start(conf *config.Config, eventHandler handlers.Handler) {
 		go c.Run(stopCh)
 	}
 
+	if conf.Resource.ConfigMap {
+		informer := cache.NewSharedIndexInformer(
+			&cache.ListWatch{
+				ListFunc: func(options meta_v1.ListOptions) (runtime.Object, error) {
+					return kubeClient.CoreV1().ConfigMaps(meta_v1.NamespaceAll).List(options)
```
Comment (missedone) [inline pkg/controller/controller.go]: i'm also thinking of make the namespace parameterized so that watch event on interested namespace only

Comment (missedone) [reply pkg/controller/controller.go]: see #115

**Comment by jbianquetti-nami** [comment]
@jjo can you take a quick review on this one if you have the chance?

**Comment by tylerauerbeck** [comment]
@jbianquetti-nami LGTM

**Pull request merged**